src/components/textfieldSlider/js/TextfieldSlider.js

```diff
@@ -39,6 +38,23 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
                 }
             }
         },
+        contextAwareness: {
+            sliderVariety: {
+                checks: {
+                    nativeHTML: {
+                        contextValue: "{fluid.prefsWidgetType}",
+                        equals: "nativeHTML",
+                        gradeNames: "fluid.textfieldSlider.nativeHTML"
+
+                    },
```

Comment:
Do we actually have to check for native? Could we set the native slider as the default in the component and just override it with the jQuery one if the contextValue is set to jQueryUI?